src/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const db = require("../src/db");
+
+const app = express();
+const port = process.env.PORT || 4000;
+
+const cors = require("cors");
+const corsMiddleWare = cors();
+
+app.use(corsMiddleWare);
+
+const bodyParser = require("body-parser");
+const bodyParserMiddleWare = bodyParser.json();
+
+app.use(bodyParserMiddleWare);
+
+app.get("/ping", (request, response) => {
+  response.send("are you there?");
+});
+
+const userRoutes = require("../src/user/router");
+const studioRoutes = require("../src/studio/router");
+const projectRoutes = require("../src/project/router");
+const imageRoutes = require("../src/image/router");
+const workshopRoutes = require("../src/workshop/router");
```

Comment:
All imports/requires go at the top of the file, and as a rule of thumb we do:
1. Libraries and modules (express, cors, etc)
2. Your own files (routers/models)